packages/components/src/RadarChart/RadarChart.component.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+export { Radar, PolarAngleAxis } from 'recharts';
```

Comment:
eslint yell at me, exports should be  after imports

Reply:
order updated.